### DISCUSSION TITLE
[Innspill til komponent]: Integrer en option for klarspråkhjelp i Textarea

### DISCUSSION THREAD

**Discussion by sarahbrodwall** [navikt/Designsystemet#1812] [General]
### Tilbakemelding

@karihamnes nevnte at de har integrert https://navikt.github.io/spraksjekk/ i fritekstfeltet saksbehandlerne bruker for å skrive custom tekst til innbyggerne.  Hun foreslår at dette hadde vært en smart feature å tilby som en del av textarea-komponenten i designsystemet.  Er dette noe vi kan få til?

**Comment by lmfaole** [comment]
Interessant! Hvordan har de gjort det? Har du noen skjermbilder? 👀

**Comment by karihamnes** [comment]
![image](https://user-images.githubusercontent.com/61583515/218114323-11eacecc-5736-45c4-b19e-afbde4d58b16.png)
![image](https://user-images.githubusercontent.com/61583515/218114444-8c0bf82a-396d-4585-be2a-0206ba8ea46e.png)

Vi har foreløpig bare implementert 2 av kildene. Vi har laget en switch for å skru språkhjelpa på/av, fordi ikke alle liker å få tilbakemeldinger fortløpende. Dette er i pilot ved ett NAV-kontor. Av tilbakemeldinger vi har fått:
- godt mottatt, fungerer fint
- men LIKS kan være litt vanskelig å forstå, fordi det står ting som "Vanskelig å lese", men ikke "for hvem"? Og da må man konsultere andre kilder for å sette seg inn i det. 
- De har sett tendenser til at de tenker litt "spill", forsøker å få tallet så lavt som mulig, uten å vurdere hvem de skriver for, eller forsøker å få bort alle "feil", selv om det kan være grunner til å skrive på den spesifikke måten
- Det er gjort tilpasninger av koden for å kunne bruke den i fagsystemsonen/i vår applikasjon, som noen av utviklerne våre sikkert kan fortelle mer om, f.eks. @Mathiamu eller @klaramargrethehelgemo 
- @eilifjohansen hjalp oss da vi gjorde denne jobben, og vi hadde en slags "hackathon" for å implementere første versjon til denne piloten
- Vi ønsker å få den implementert i en mer komplett versjon i vår løsning, men mener dette ville være en veldig viktig komponent å tilby til bruk i alle fagsystemer med inputfelter der det skrives fritekst som skal leses av innbyggere, og håper derfor på at dette er noe som kanskje designsystemet trigger på! Vil gi praktisk klarspråkhjelp med virkelig verdi, i kontekst av alle saksbehandlingsløsningene som utvikles.

**Comment by sjur-gr** [comment]
Veldig kult! Ser helt klart at dette kan være til hjelp. Til å starte med kan vi kanskje legge til et avsnitt i Textarea-dokumentasjonen som sier litt om klarspråk-komponenten og lenker ut til den. På sikt kan dette dokumenteres som et eget mønster, med kodesnutter og forklaringer. Textarea-komponenten er det best at vi holder så slank og spissa som mulig siden den blir brukt ganske mange steder hvor en sånn integrasjon ikke gir fult så mye mening. @eilifjohansen ligger klarspråk på et åpent repo?

**Comment by karihamnes** [comment]
Kan nevnes at også [team DAB i PO arbeidsoppfølging](https://teamkatalog.nav.no/team/1ad2c9ea-3221-4666-93f3-fe6f7cae94ef) er interessert i å implementere Klarpråkhjelpen på interne flater og knytte til samtalereferater som veiledere skriver i aktivitetsplanen i Modia arbeidsrettet oppfølging. @eilifjohansen
